server/src/utils/clan/updateClanPoints.ts

```diff
@@ -1,4 +1,4 @@
-import { getBruteGoldValueFromLevel, getBruteGoldValueFromRanking } from '@labrute/core';
+import { getBruteGoldValueFromLevel, getBruteGoldValueFromRanking } from '@labrute/core/src/brute/getBruteGoldValue.js';
```

Comment:
Also, this is not the correct way to import from core, it should be the same as before, @labrute/core